docs/README.md

```diff
@@ -140,14 +136,27 @@ env:
 
 Check out [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/) for more information about injecting environment variables into applications!
 
-### Run as Non-Root
+### Configuring Lookback
 
-Beginning with version 2.2 of the agent, the Dockerfile supports running the agent as a non-root user. This behavior; however, is not the default as to make the new version backwards compatible with old versions of the agent yamls. Make the following changes to your DaemonSet to run the agent as non-root:
+The lookback strategy determines how the agent handles existing container logs on startup. This strategy is determined by `LOGDNA_LOOKBACK`. There's a limited set of valid values for this option:
 
-Add two new fields, `runAsUser` and `runAsGroup`, to the `securityContext` section found in the `logdna-agent` container in the `logdna-agent` DaemonSet [`spec.template.spec.containers.0.securityContext`]:
+* `start` - Always start at the beginning of the files.
+* `smallfiles` - If the file is less than 8KiB, start at the beginning. Otherwise, start at the end.
+* `none` - Always start at the end of the files.
+* __Note:__ The default option is `smallfiles`
 
-```yaml
-securityContext:
-  runAsUser: 5000
-  runAsGroup: 5000
-```
+### Configuring Journald
+
+If the agent pods have access to journald log files or directories, monitoring can be enabled on them with the `LOGDNA_JOURNALD_PATHS`. Common values include `/var/log/journald` and `/run/systemd/journal`. To specify both, use a comma separated list: `/var/log/journald,/run/systemd/journal`.
+
+Take a look at enabling journald monitoring for [Kubernetes](KUBERNETES.md#collecting-node-journald-logs) or [OpenShift](OPENSHIFT.md#collecting-node-journald-logs).
+
+### Configuring Events
+
+Kubernetes and OpenShift events are by default automatically captured by the agent. This feature can be controlled by the `LOGDNA_LOG_K8S_EVENTS` option with only two valid values:
+
```

Comment:
Consider this instead, which highlights the "by default"part and avoids passive voice:

"By default, the LogDNA agent captures Kubernetes events (and Openshift events, as well, since Openshift is built on Kubernetes clusters). 

To control whether the LogDNA agent collects Kubernetes events, configure the `LOGDNA_LOG_K8S_EVENTS` option using one of these two values:"
+++++

Also, we need to tell readers WHERE to configure the `LOGDNA_LOG_K8S_EVENTS` option.

Reply:
fixed

Reply:
Yeh, it's in the k8s yaml env section, I think this should be fairly intuitive based on how the rest of the configuration is done